bundles/org.openhab.binding.samsungtv/src/main/resources/OH-INF/config/config.xml

```diff
@@ -5,14 +5,22 @@
 	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
 
 	<config-description uri="thing-type:samsungtv:tv">
+		<parameter-group name="Cloud Connection">
+			<label>Smartthings Cloud connection (Optional 2016 and later TV's)</label>
```

Comment:
Some new labels in this (and other) xml are a bit lengthy. Are you able to check them and shorten them a bit?
_<25 chars, max 2-3 words) and capitalized_

Reply:
Labels shortened.

Reply:
I see, great, now the labels are adjusted you also need to re-run the i18n plugin.

Reply:
I think everything is done now.